python/huggingfaceserver/README.md

```diff
@@ -187,3 +187,37 @@ curl -H "content-type:application/json" -v localhost:8080/openai/v1/chat/complet
 
 {"id":"cmpl-87ee252062934e2f8f918dce011e8484","choices":[{"finish_reason":"length","index":0,"message":{"content":"<generated_response>","tool_calls":null,"role":"assistant","function_call":null},"logprobs":null}],"created":1715353461,"model":"gpt2","system_fingerprint":null,"object":"chat.completion","usage":{"completion_tokens":30,"prompt_tokens":3,"total_tokens":33}}
 ```
+
+### KServe HuggingFace vLLM Runtime Support for CPU
+vLLM powered by OpenVINO supports all LLM models from vLLM supported models list and can perform optimal model serving on all x86-64 CPUs with, at least, AVX2 support. You can find more information [here](https://docs.vllm.ai/en/v0.6.1/getting_started/openvino-installation.html). To run the vLLM engine on a CPU, a separate vLLM package is required, which entails creating a different Hugging Face server Docker image. The Docker image supporting the vLLM runtime on GPUs has a '-gpu' suffix in its tag (e.g., kserve-huggingfaceserver:v0.14.0-gpu), while the image for CPU support does not include the suffix (e.g., kserve-huggingfaceserver:v0.14.0). The runtime image to be used will be determined based on whether the InferenceService specifies nvidia.com/gpu in its resource requirements.
```

Comment:
This naming convention might be confusing for people who were used to using huggingfaceserver-vLLM without any tag as GPU supported image

Reply:
This is currently being followed for TFServing and Torchserve predictors, based on whether the InferenceService specifies nvidia.com/gpu in its resource requirements "-gpu" will be added by controller if not present at end of tag